app/controllers/api_controller/providers.rb

```diff
@@ -5,9 +5,14 @@ module Providers
     CREDENTIALS_ATTR  = "credentials"
     AUTH_TYPE_ATTR    = "auth_type"
     DEFAULT_AUTH_TYPE = "default"
-    ENDPOINT_ATTRS    = %w(hostname ipaddress port)
+    ENDPOINT_ATTRS    = %w(hostname ipaddress port connection_configurations_array).freeze
```

Comment:
is this really an Endpoint attribute ?  shouldn't that be coming to us from ExtManagementSystem.

why are we exposing connection_configurations_array (it's a virtual_total), shouldn't we be exposing connection_configurations instead ?